lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -5,6 +5,39 @@ module Sidekiq
     class QueueLatency
       extend Forwardable
 
+      DEFAULT_LATENCY_VALUES = {
```

Comment:
I think this should live in `Sidekiq::Settings ` (it's weird that Settings has to ask `QueueLatency` for the list of queues).